paths/__init__.py

```diff
@@ -15,7 +15,7 @@
     KelkRonso,
     SeymourFlux,
 )
-from paths.guadosalam import GuadoSkip, GuadoStoryline
+from paths.guadosalam import GuadoStart, GuadoSkip, GuadoStoryline
```

Comment:
`GuardoStart` needs to be added to the `__all__` list below. Otherwise some tools will not know that this is here. It's not a huge deal with how we are doing things + what tools we are using, but it's more correct if it's in that list.

Reply:
Added.